plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -239,6 +243,28 @@ export default class ChatMessage {
     return this.manager?.messages?.objectAt?.(this.index + 1);
   }
 
+  addMentionedUser(user) {
+    this.mentionedUsers.set(user.id, user);
+  }
+
+  async ensureMentionsLoaded(ignoreFailure) {
+    try {
+      const notLoaded = this.#notLoadedMentions;
+      if (notLoaded.length > 0) {
+        const users = await this.usersApi.lookupUsers(notLoaded);
+        users.forEach((user) => this.addMentionedUser(user));
+        this.incrementVersion();
+      }
+    } catch (e) {
+      if (ignoreFailure) {
+        // eslint-disable-next-line no-console
+        console.warn("Cannot load mentioned users", e);
+      } else {
+        throw e;
+      }
```

Comment:
We always call it with `ignoreFailure: true` so I would make it the default.

Reply:
I agree with that one, if `ignore failure` is the happy path, it should be the default

Reply:
This was added after this suggestion https://github.com/discourse/discourse/pull/21935#discussion_r1235445298. When addressing that, I didn't notice we're ending up with an argument we effectively don't use, my bad.

I fixed it in b31580ec08add346c4c5d12eb5301e744dd371c8, I switched it back to the previous version, but made it simpler to address @jjaffeux's concern in that old comment.